src/main/java/com/wrapper/spotify/model_objects/specification/AlbumSimplified.java

```diff
@@ -283,53 +330,62 @@ public AlbumSimplified createModelObject(JsonObject jsonObject) {
       }
 
       return new AlbumSimplified.Builder()
-              .setAlbumType(
```

Comment:
I'd like this indentation to be kept. Is IntelliJ's indentation even compatible with setting indentation as in https://github.com/thelinmichael/spotify-web-api-java/pull/179?